blocks/into/__subtitle/into__subtitle.css

```diff
@@ -0,0 +1,23 @@
+.into__subtitle{
+    display: flex;
+    width: 300px;
+    height: 37px;
```

Comment:
You can use padding top and bottom instead of giving height to text element. Specify height only when it is really needed. It breaks the responsiveness of the website.